WithSecure/withsecure/response_action.py

```diff
@@ -0,0 +1,50 @@
+from threading import Event
+from typing import Any
+
+from sekoia_automation.action import Action
+
+from withsecure.client import ApiClient
+from withsecure.constants import API_RESPONSE_ACTIONS_URL, API_TIMEOUT
+
+
+class ResponseAction(Action):
+    def run(self, arguments: Any) -> Any:
+        raise NotImplementedError()
+
+    def _execute_operation_on_device(
+        self, action_type: str, target: str, organization_id: str, parameters: dict[str, Any] | None = None
+    ) -> Any:
+        self.log(
+            f"Execute the action type '{action_type}' on device '{target}'",
+            level="debug",
+        )
+
+        request_payload: dict[str, Any] = {
+            "type": action_type,
+            "organizationId": organization_id,
+            "targets": [target],
```

Comment:
According to some tests I made with the API, the field `parameters` is required (even empty)
```suggestion
            "targets": [target],
            "parameters": {},
```